CONFIGURATION.md

```diff
@@ -76,3 +76,14 @@ Users of nginx/apache must ensure to have matching CORS configurations.
 * `PAGINATION_ENABLED`: whether the pagination is enabled (default: `True`)
 * `PAGINATION_DEFAULT_PAGE_SIZE`: the default page size if no query param (`page_size`) is given (default: `100`)
 * `PAGINATION_MAX_PAGE_SIZE`: the max value of the page size query param (`page_size`) (default: `1000`)
+
+## Email error handler
+* `ENABLE_ADMIN_EMAIL_LOGGING`: enable Django to send email to admins on errors (default: `False`)
+* `SERVER_MAIL`: the email address that error messages come from
+* `EMAIL_HOST`: the host to use for sending email (default: `localhost`)
+* `EMAIL_PORT`: port to use for the SMTP server (default: `25`)
+* `EMAIL_HOST_USER`: username for the SMTP server(default: "")
+* `EMAIL_HOST_PASSWORD`: password for the SMTP server user (default: "")
+* `EMAIL_USE_TLS`: whether to use an implicit TLS (secure) connection when talking to the SMTP server  (default: `False`)
+
+If either `EMAIL_HOST_USER` or `EMAIL_HOST_PASSWORD` is empty, Django won't attempt authentication.
```

Comment:
Could you add the documentation for the `ADMINS` env var here as well, including a description that those will receive the error mails and an example of a valid value (e.g. `ADMINS=John Doe <john.doe@example.com>,Jane Doe <jane.doe@example.com>`)

Reply:
added documentation for `ADMINS`